File: stock-market-service/src/main/java/rs/edu/raf/dto/StockHistoryInfoDTO.java
package rs.edu.raf.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
public record StockHistoryInfoDTO(
    @JsonProperty("1. open")
    Double open,
    @JsonProperty("2. high")
    Double high,
    @JsonProperty("3. low")
    Double low,
    @JsonProperty("4. close")
    Double close,
    @JsonProperty("5. volume")
    Long volume
) { }

File: stock-market-service-kotlin/src/main/kotlin/org/example/stockmarketservicekotlin/dto/StockHistoryInfoDTO.kt
package org.example.stockmarketservicekotlin.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class StockHistoryInfoDTO(
    @JsonProperty("1. open")
    val open: Double,
    @JsonProperty("2. high")
    val high: Double,
    @JsonProperty("3. low")
    val low: Double,
    @JsonProperty("4. close")
    val close: Double,
    @JsonProperty("5. volume")
    val volume: Long
)
